ui-participant/src/providers/UserProvider.tsx

```diff
@@ -99,6 +103,9 @@ export default function UserProvider({ children }: { children: React.ReactNode }
         enrollees: updatedEnrollees
       }
     })
+    if (afterFn) {
+      window.setTimeout(afterFn, 0)
+    }
```

Comment:
To make the interface of this function a bit more conventional, I'd suggest returning a Promise instead of taking a callback.

```suggestion
    return new Promise(resolve => {
      window.setTimeout(resolve, 0)
    })
```

And call it like:
```ts
updateEnrollee(...).then(() => {
  ...
})
```